packages/migrations/src/clickhouse-actions/014-audit-logs-access-token.ts

```diff
@@ -0,0 +1,10 @@
+import type { Action } from '../clickhouse';
+
+export const action: Action = async exec => {
+  await exec(`
+    ALTER TABLE "audit_logs"
+      ADD COLUMN IF NOT EXISTS
+        "access_token_id" String CODEC(ZSTD(1))
```

Comment:
nit(?): if we only store id, it might not have enough info when the token is deleted? should it be also a name?

Reply:
We store more information about the token within the metadata column. See https://github.com/graphql-hive/console/pull/6738/files#diff-2ef2e8ba27a98af0972ca875f10b24fe905bcc7d31bdb1791df098ff42fa0872R47-R53